docs/source/configuring-slack-for-spidermon.rst

```diff
@@ -0,0 +1,31 @@
+.. _steps:
```

Comment:
These anchors are global. You should use an ID that can be understood from anywhere in the documentation, e.g. `configuring-slack`, `slack-bot`…

Reply:
Implemeted.